test/controllers/api/v2/compute_resources_controller_test.rb

```diff
@@ -476,4 +476,41 @@ def teardown
       assert_response :unprocessable_entity, "Can create libvirt compute resource with invalid name"
     end
   end
+
+  context 'ovirt' do
+    setup do
+      @simple_json = {
```

Comment:
This could be defined via `let(:simple_json){ {} }`

Reply:
Have you tried putting the `let` outside the `setup` block?

Reply:
damnn no, my bad